addons/l10n_br_website_sale/models/website.py

```diff
@@ -0,0 +1,11 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    def _display_partner_b2b_fields(self):
+        """ Uruguay localization must always display b2b fields. """
```

Comment:
@kcv-odoo just found this a bit strange, as it seems to refer to UY but it is inside the BR file `addons/l10n_br_website_sale/models/website.py` are you sure this should be like this?

Reply:
Should be fine now! forgot to update country code when copy/pasting method :man_facepalming:

Reply:
Great! Thanks! Now I see the `ID Type` and `ID Number` when on a B2C website in BR.

Reply:
@kcv-odoo just as a NTH, when B2B is off, we could hide the "Company Name" and keep the `ID Number` and `ID Type`. Company name is indeed only needed for B2B and not something required in BR for B2C